### PR TITLE
Endpoints examples

### PR DESCRIPTION
**Pull request by xuebingjie1990**
update bedfiles and bedset schema.
allowed bedhost dynamically define and set labels for file/image choices for the drop-down lists.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format.
 
+## [0.1.1] - 2021-04-15
+### added
+- added new feilds in the bedfiles and bedsets schema
```
Comment (stolarczyk) [inline docs/changelog.md:7]: typo

File: bbconf/schemas/bedfiles_schema.yaml [hunk 2]
```diff
@@ -66,31 +68,43 @@ promotercore_percentage:
...
 neighbor_distances:
   type: image
+  label: Neighbor_distance
   description: Distance between neighbor regions
 open_chromatin:
   type: image
+  label: Open_chromatin
   description: Cell specific enrichment for open chromatin plot
+genome:
+  type: object
```
Comment (stolarczyk) [inline bbconf/schemas/bedfiles_schema.yaml:106]: why is this an object, not a string?

**Comment by stolarczyk** [comment]
do the labels need to start with a capital letter?

**Comment by xuebingjie1990** [comment]
> do the labels need to start with a capital letter?

no, I just thought it would look better in the list. 

Screenshot from 2021-04-15 11-30-17

File: bbconf/schemas/bedfiles_schema.yaml (hunk 2 above)
Comment (xuebingjie1990) [reply bbconf/schemas/bedfiles_schema.yaml:106]: > why is this an object, not a string?

it should be string for this version. I'll change it back.

**Comment by stolarczyk** [comment]
In my opinion, if you you care about the looks and it's intended to be displayed in the UI then it's goof to format it like this: `'String string'`, if it you want it to be internal and computer readable then: `'string_string'`

File: bbconf/schemas/bedfiles_schema.yaml (hunk 2 above)
Comment (xuebingjie1990) [reply bbconf/schemas/bedfiles_schema.yaml:106]: > why is this an object, not a string?

for the current database, there is no  `genome` field in the bedfile table yet. the update make on `bedstats`  and `bedbuncher` added/changed the `genome` field with the refgenie digest, something like
```
genome: {
     alias: hg38
     digest: 2230c535660fb4774114bfa966a62f823fdb6d21acf138d4
}
```

should remove https://github.com/databio/bbconf/blob/539126e288a1f7d0dfd8e211d1492c2546e8af39/bbconf/schemas/bedfiles_schema.yaml#L105-L107 for this bbconf release?

**Comment by xuebingjie1990** [comment]
> In my opinion, if you you care about the looks and it's intended to be displayed in the UI then it's good to format it like this: `'String string'`, if it you want it to be internal and computer readable then: `'string_string'`

I think the `label` field is only used for UI, at least for now.

File: bbconf/schemas/bedfiles_schema.yaml (hunk 2 above)
Comment (stolarczyk) [reply bbconf/schemas/bedfiles_schema.yaml:106]: well, the point of this release is to make the prematurely released bedhost work, so set the genome type to what's needed for metadata served with the main instance of bedbase

**Review by stolarczyk — APPROVED** [review]

**Pull request merged**